public/data/meet-the-talent.json

```diff
@@ -399,6 +430,39 @@
           "name": "X/Twitter"
         }
       ]
+    },
+    {
+      "fullName": "Divya Dave",
+      "location": "Ahmedabad, India",
+      "bio": "With 4 years of experience, I bring a thoughtful approach to development, asking the right questions to create impactful, user-focused mobile applications that prioritize seamless experiences.",
+      "email": "divyadave1205@gmail.com",
+      "phone": "+91-9601808779",
+      "imageUrl": "https://github.com/user-attachments/assets/5d5fa3f9-4f58-4fa5-a10d-80e4d2ab94a1",
```

Comment:
imageUrl doesn't look like a publicly accessible. Can you please verify it and update the PR? It should basically be direct reference of image.